### ISSUE TITLE
[TSDoc> Playground]: Ensures iframe and frame elements have an accessible name for the frame under playground.

### ISSUE THREAD

**Issue by vagpt** [microsoft/rushstack-websites#250]
**Test Environment:**
Version: Version 24H2 (OS Build 26231.5000)
Browser Version: 127.0.2638.2 (Official build) dev (64-bit)
URL: https://tsdoc.org/play/
Tool: Accessibility Insight For Web

**Repro Steps:**
1. Open the above URL on the EDGE Dev browser.
2. Press the tab key and navigate to the 'PlayGround' option and press enter key to activate it.
3. Run the tool and observe the issue.

**Actual Result:**
On the playground screen, <iframe> and <Frame> does not have accessible name.

**Expected Result:**
Ensures iframe and frame elements have an accessible name for the frame under playground.

**Element path:**
iframe

**Snippet:**
<iframe src="https://microsoft.github.io/tsdoc/" style="flex-grow: 1;"></iframe>

**How to fix:** 
Fix any of the following:
  Element has no title attribute
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element's default semantics were not overridden with role="none" or role="presentation"

**User Impact:**
It will impact the screen reader user as they will not able to get to know the name of the frame if the name is not defined for the iframe.

**WCAG Reference**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment.

![Ensures iframe and frame elements have an accessible name for the frame under playground page](https://github.com/microsoft/rushstack-websites/assets/95913748/9b4bb9e0-3a9e-466e-b95d-88acd839ba30)

![Ensures iframe and frame elements have an accessible name for the frame under playground](https://github.com/microsoft/rushstack-websites/assets/95913748/28248f62-4965-4481-b649-b367ee35844d)
